srv/modules/runners/push.py

```diff
@@ -72,6 +72,16 @@ def proposal(filename = "/srv/pillar/ceph/proposals/policy.cfg", dryrun = False)
     pillar_data.output(common)
     return True
 
+def __create_dirs(path, root):
```

Comment:
I haven't found it yet.  I suspect we could have something live under /usr/lib/python2.7/site-packages/salt, but that does not help with packaging.  I suppose the closest I have come with a common function for runners is a module.  This might seem strange or abusive, but we have more than this create_dirs example that is common across runners.